src/gui/widgets/listbox.cpp

```diff
@@ -615,10 +615,19 @@ void listbox::set_active_sorting_option(const order_pair& sort_by, const bool se
 	// TODO: should this be moved to a public header_grid() getter function?
 	grid& header_grid = find_widget<grid>(this, "_header_grid", false);
 
-	selectable_item& wgt = find_widget<selectable_item>(&header_grid, "sort_" + std::to_string(sort_by.first), false);
-	wgt.set_value(static_cast<int>(sort_by.second));
+	selectable_item* w =
+		find_widget<selectable_item>(&header_grid, "sort_" + std::to_string(sort_by.first), false, false);
 
-	order_by_column(sort_by.first, dynamic_cast<widget&>(wgt));
+	if(!w) {
```

Comment:
Why did you add this check? Unless this condition can be triggered by UMC, it's better to crash than to silently fail to sort because it alerts us about the problem.